docker/.env.example

```
@@ -0,0 +1,31 @@
+GORGONE_TAG=
+SNMPTRAPD_TAG=
+CENTREONTRAPD_TAG=
+VMWARE_TAG=
+
+# === Runtime ===
+TZ=Europe/Paris
+DEBUG=false
+
+# === Poller identity ===
+NAME=my-poller
+GORGONE_UID=       # Poller UID from Central (required)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)